[2025-01-17T16:57:53.145+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_process_with_excel_report.transform_data scheduled__2025-01-16T00:00:00+00:00 [queued]>
[2025-01-17T16:57:53.166+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_process_with_excel_report.transform_data scheduled__2025-01-16T00:00:00+00:00 [queued]>
[2025-01-17T16:57:53.168+0000] {taskinstance.py:1361} INFO - Starting attempt 16 of 16
[2025-01-17T16:57:53.275+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform_data> on 2025-01-16 00:00:00+00:00
[2025-01-17T16:57:53.441+0000] {standard_task_runner.py:57} INFO - Started process 1623 to run task
[2025-01-17T16:57:53.543+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_process_with_excel_report', 'transform_data', 'scheduled__2025-01-16T00:00:00+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/db_to_xls.py', '--cfg-path', '/tmp/tmpgd9flsg8']
[2025-01-17T16:57:53.563+0000] {standard_task_runner.py:85} INFO - Job 52: Subtask transform_data
[2025-01-17T16:57:53.881+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_process_with_excel_report.transform_data scheduled__2025-01-16T00:00:00+00:00 [running]> on host 4e4dfd1b49e6
[2025-01-17T16:57:54.066+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_process_with_excel_report' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='16' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-16T00:00:00+00:00'
[2025-01-17T16:57:54.765+0000] {db_to_xls.py:124} INFO - Transformation completed and reports saved.
[2025-01-17T16:57:54.768+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-01-17T16:57:54.851+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_process_with_excel_report, task_id=transform_data, execution_date=20250116T000000, start_date=20250117T165753, end_date=20250117T165754
[2025-01-17T16:57:54.932+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-17T16:57:54.997+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
